src/render-table.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+// {
+//   attributes: { ... },
```

Comment:
We should add
```js
caption: {
  attributes: { ... },
  content: { ... },
},
```
In order to be aligned with [HTML standard](https://www.w3.org/TR/html/tabular-data.html#the-table-element).